helm/applications/skaha/values.yaml

```diff
@@ -117,6 +117,18 @@ deployment:
       #        priorityClass: "user-sessions-med"
       queue: {}
 
+      # This is a list of tolerations that will be added to the Pod spec of the User Sessions.
```

Comment:
We should add more documentation, especially in a `README` that these are two different tolerations. This could be misconfigured very easily.